ISSUE_TRIAGE.rst

```diff
@@ -0,0 +1,93 @@
+- | ``needs design``: This feature is large or tricky enough that we think it warrants a design doc 
+  | and review before someone begins implementation.
+- | ``needs review``: Use this label for issues that need a more detailed design review or pull 
+  | requests ready for review (all questions answered, PR updated if requests have been addressed, 
+  | tests passing).
+- ``help wanted``: We would like community help for this issue.
+- ``good first issue``: This would make a good first issue.
+
+
+Assign priority
+#######
+
+You should assign a priority to each issue you triage. We use `kubernetes-style <https://github.com/
+kubernetes/community/blob/master/contributors/guide/issue-triage.md#define-priority>`_ priority 
+labels.
+
+- | ``priority/critical-urgent``: This is the highest priority and should be worked on by
+  | somebody right now. This should typically be reserved for things like security bugs, 
+  | regressions, release blockers.
+- | ``priority/important-soon``: The issue is worked on by the community currently or will 
+  | be very soon, ideally in time for the next release.
+- | ``priority/important-longterm``: Important over the long term, but may not be staffed or
+  | may need multiple releases to complete. Also used for things we know are on a 
+  | contributor's roadmap in the next few months. We can use this in conjunction with 
+  | ``help wanted`` to mark issues we would like to get help with. If someone begins actively
+  | working on an issue with this label and we think it may be merged by the next release, change
+  | the priority to ``priority/important-soon``.
+- | ``priority/backlog``: We believe it is useful but don’t see it being prioritized in the 
+  | next few months. Use this for issues that are lower priority than ``priority/important-longterm``. 
+  | We welcome community members to pick up a ``priority/backlog`` issue, but there may be some 
+  | delay in getting support through design review or pull request feedback. 
+- | ``priority/awaiting-more-evidence``: Lowest priority. Possibly useful, but not yet enough
+  | support to actually get it done. This is a good place to put issues that could be useful but 
+  | require more evidence to demonstrate broad value. Don’t use it as a way to say no. 
+  | If we think it doesn’t fit in MLflow, we should just say that and why.
+
+Label relevant areas
+#######
+
+Assign one more labels for relevant component or interface surface areas, languages, or 
+integrations. As a principle, we aim to have the minimal set of labels needed to help route issues
+and PRs to appropriate contributors. For example, a ``language/python`` label would not be
+particularly helpful for routing issues to committers, since most PRs involve Python code.
+``language/java`` and ``language/r`` make sense to have, as the clients in these languages are 
+differ from the python client and aren't maintained by many people. As with process labels, we
```

Comment:
@pogil One more small typo that I missed (or maybe that I introduced via suggestion during the last round of reviews):

```suggestion
``language/java`` and ``language/r`` make sense to have, as the clients in these languages differ from the Python client and aren't maintained by many people. As with process labels, we
```